src/commands/part.cpp

```diff
@@ -27,11 +27,8 @@ int command_part(Message &message, Dependencies &deps)
 				if (channel.users.has(message.peer.getFd()))
 				{
 					channel.remove(channel.users[message.peer.getFd()]);
-					if (!message.arguments[1].empty())
-					{
-						Message reply(message.peer, message.arguments[1]);
-						channel.sendMessage(reply);
-					}
```

Comment:
What if the message is empty then? Is the reason empty too ? Should the message not include the second argument in his buffer ? Does irssi have a strange behavior ?

Reply:
The reason is optional. In which case the message may be empty?
Irssi is working fine, his medication help him to stay happy and calm. Thank you for caring.